test/parallel/test-runner-run.mjs

```diff
@@ -4,9 +4,15 @@ import { join } from 'node:path';
 import { describe, it, run } from 'node:test';
 import { dot, spec, tap } from 'node:test/reporters';
 import assert from 'node:assert';
+import tmpdir from '../common/tmpdir.js';
 
 const testFixtures = fixtures.path('test-runner');
 
+// This is needed, because calling run() with no files will
+// load files matching from cwd, which includes all content of test/
+tmpdir.refresh();
+process.chdir(tmpdir.path);
```

Comment:
Won't this break running the test suite in worker threads?

Reply:
Without this, I would need to fix https://github.com/nodejs/node/issues/53867 as well in this PR.

Let's see if it actually breaks something in CI.

Reply:
It looks like it did break in the CI (https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37045/). The issue is that the Node test suite is run within a worker thread in the CI. Any test that uses an API that is unsupported in worker threads needs to be skipped there.

Reply:
Ok. I'll need to add a `cwd` parameter to `run()`, otherwise it won't work.

Reply:
I would hold off on adding a `cwd` parameter for now, but I do foresee us adding it in the future. I'm just not sure if it will break anything until the necessary refactoring happens. Another option is to spawn a child process with the correct cwd.